### PR TITLE
[Form] Document "class constraints" for data class-less forms

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/18969#issuecomment-1783227431

### CODE REVIEW

File: form/without_class.rst [hunk 1]
```diff
@@ -123,3 +126,56 @@ but here's a short example::
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another possibility is to add the constraints at the class level.
+This can be done by setting the ``constraints`` option in the
+``configureOptions()`` method::
+
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
+    use Symfony\Component\Form\FormBuilderInterface;
+    use Symfony\Component\Validator\Constraints\Length;
+    use Symfony\Component\Validator\Constraints\NotBlank;
```
Comment (OskarStark) [inline form/without_class.rst:141]: Ue statement for OptionsResolver is missing

**Review by OskarStark — CHANGES_REQUESTED** [review]

File: form/without_class.rst [hunk 2]
```diff
@@ -123,3 +126,56 @@ but here's a short example::
...
+                'lastName' => [
+                    new NotBlank(),
+                    new Length(['min' => 3]),
+                ],
+            ],
+        ])
+        ->add('firstName', TextType::class)
+        ->add('lastName', TextType::class)
+        ->getForm()
+    ;
```
Comment (OskarStark) [inline form/without_class.rst]:
```suggestion
        ->getForm();
```

File: form/without_class.rst [hunk 3]
```diff
@@ -123,3 +126,56 @@ but here's a short example::
...
+    use Symfony\Component\Form\FormBuilderInterface;
+    use Symfony\Component\Validator\Constraints\Length;
+    use Symfony\Component\Validator\Constraints\NotBlank;
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('firstName', TextType::class)
+            ->add('lastName', TextType::class)
+        ;
```
Comment (OskarStark) [inline form/without_class.rst]:
```suggestion
            ->add('lastName', TextType::class);
```

File: form/without_class.rst (hunk 1 above)
Comment (alexandre-daubois) [reply form/without_class.rst:141]: Good catch 👍

File: form/without_class.rst (hunk 2 above)
Comment (alexandre-daubois) [reply form/without_class.rst]: Should we update other examples to follow this code style?

Comment (OskarStark) [reply form/without_class.rst]: Not sure, I just want to make it more compact.

@javiereguiluz WDYT?

**Comment by OskarStark** [comment]
Thank you Alexandre.

**Pull request merged**